tests/vufind.php-cs-fixer.php

```diff
@@ -19,12 +19,21 @@
     'concat_space' => ['spacing' => 'one'],
     'ereg_to_preg' => true,
     'get_class_to_class_keyword' => true,
+    'global_namespace_import' => [
+        'import_constants' => null,
```

Comment:
Since null is the default value for import_constants, would it be better to omit it here? Or do you think it might be worth changing to true in a subsequent PR?

Reply:
sure.